### PR TITLE
include ssl port in AppDashboard /apps/json route

### PR DESCRIPTION
**Pull request by selfcommit**

### CODE REVIEW

File: AppDashboard/lib/app_dashboard_helper.py [hunk 1]
```diff
@@ -319,6 +319,37 @@ def get_app_port(self, appname):
         "that it runs on.".format(appname))
 
 
+  def get_ssl_port(self, appname):
+    """ Queries the UserAppServer to learn the https port the named application.
```
Comment (menivaitsi) [inline AppDashboard/lib/app_dashboard_helper.py]: Please change "learn the https port the named application." to "learn the https port of the named application.".

**Comment by menivaitsi** [comment]
CI build: https://ocd.appscale.com:8080/job/Daily%20Build/1155/

**Comment by selfcommit** [comment]
rake now passes all tests successfully.

**Comment by menivaitsi** [comment]
Thanks for updating @selfcommit 

Latest build: https://ocd.appscale.com:8080/job/Daily%20Build/1162/

**Pull request merged**